setup.py

```diff
@@ -0,0 +1,194 @@
+"""
+This is the setup module for the example project.
```

Comment:
@zenine07 - Reminder that you should look for any instances of `example` or `skeleton-python-library` in this file and replace them appropriately.

Reply:
@dav3r Fixed those references https://github.com/cisagov/domain-manager-api/pull/194/commits/e1fb8c3b4d4c017cc9263e8cb9b68695d99ce056. Let me know if everything is looking good now.